backend/audios.js

```diff
@@ -0,0 +1,40 @@
+const express = require('express');
```

Comment:
Really love how you're organizing your file structure with having routes for each schema correspond to different files!